test/integration/PublishEditConsume.test.ts

```diff
@@ -146,6 +136,31 @@ function delay(interval: number) {
   }).timeout(interval + 100)
 }
 
+function uploadToIpfs(data: any): Promise<string> {
+  return new Promise((resolve, reject) => {
+    axios
+      .post(
+        'http://172.15.0.16:5001/api/v0/add',
```

Comment:
Looks like your still using a local IPFS node? I thought you were going to use our cluster?

Reply:
I used in the previous commit

Reply:
https://github.com/oceanprotocol/ocean.js/commit/676c328af44e1f617556aef0355bd2cf92c43854

Reply:
I was suggesting that we use the IPFS cluser that we're hosting

Same as what we're doing in the ocean-node tests:
https://github.com/oceanprotocol/ocean-node/blob/c5799f9d187851922a9e80a090e0e19325a06aa9/src/test/unit/storage.test.ts#L518 

There is also an IPFS CID there we're using for the get storage info test that is working fine:  `QmRhsp7eghZtW4PktPC2wAHdKoy2LiF1n6UXMKmAhqQJUA`

Reply:
The `QmRhsp7eghZtW4PktPC2wAHdKoy2LiF1n6UXMKmAhqQJUA` is working fine in the ocean-node tests. So I'd suggest we just try and use that here as well. So we skip the uploading step

Reply:
I also just checked and that asset is downloadable https://ipfs.oceanprotocol.com/ipfs/QmRhsp7eghZtW4PktPC2wAHdKoy2LiF1n6UXMKmAhqQJUA

Reply:
It is worth a try but it may not work because the provider in barge is using the local IPFS node: https://github.com/oceanprotocol/barge/blob/3abd5effa33e39d739dd51f0b3f1249475a4429c/start_ocean.sh#L115

It can take a few minutes for a new IPFS node to connect to enough peers to discover the asset, leading the test to timeout.

Reply:
yes, it did throw a timeout error